src/arrayMethodReduce.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, initialValue = 'undefined') {
+    let prevValue = initialValue;
```

Comment:
```suggestion
    let accumulator = initialValue;
```

Reply:
in official documentation it is accumulator, it's not prevValue, it's value that store result of sum of callbacks for prev values